fairseq/manifest.py

```diff
@@ -29,21 +30,39 @@ def __init__(
         self.audio_dir_paths = audio_dir_paths
         self.file_extension = file_extension
         self.path_must_contain = path_must_contain
+        self.concurrent = len(audio_dir_paths)
 
         self.out_audio_dir_path = self.output_path("audio", directory=True)
 
+        self.rqmt = {"time": 8, "cpu": 4, "mem": 4}
```

Comment:
The job does some symlinking. This should be fast. Why do we go from `mini_task=True` to 4 cpu cores for 8 hours?

Reply:
For the wav2vec 2 training, after the audio preprocessing, we had around 13 millions of small files, that is why I do not parallelism here